state-state.js

```diff
@@ -43,4 +45,54 @@ module.exports = function StateState() {
 		getParent: getParent,
 		getParentName: getParentName
 	}
+
```

Comment:
The three that are added on later are functions that take the object itself as the first argument, so they're more like true methods than functions exposing state in the closure.

It would probably be cleaner to just call those functions directly instead of accessing them as properties of the "prototypalStateObject".